content/operate/kubernetes/logs/collect-logs.md

```diff
@@ -25,17 +25,31 @@ As of version 6.2.18-3, the log collector tool has two modes:
 1. Download the latest [`log_collector.py`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/log_collector/log_collector.py) file.
 
 1. Have a K8s administrator run the script on the system that runs your `kubectl` or `oc` commands.
```

Comment:
It could be useful to also mention that the k8s CLI (kubectl / oc) needs to be [configured](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) to access the k8s cluster to be collected.

Reply:
Can also be useful to mention the required RBAC permissions for the user executing the log collector?

Reply:
> Can also be useful to mention the required RBAC permissions for the user executing the log collector?

Actually we have DOC-3998 for this.